GitTfs/Commands/InitBranch.cs

```diff
@@ -135,28 +137,48 @@ private IGitTfsRemote InitBranchSupportingRename(string tfsBranchPath, string gi
 
             _stdout.WriteLine("Branches to Initialize successively :");
             foreach (var branch in creationBranchData)
-                _stdout.WriteLine("-" + branch.TfsBranchPath + " (" + branch.RootChangeset + ")");
+            {
+                var outputMessage = "-" + branch.TfsBranchPath + " (Source C" + branch.SourceBranchChangesetId;
+
+                if (branch.TargetBranchChangesetId > -1)
+                    outputMessage += " -> Target C" + branch.TargetBranchChangesetId;
+
+                outputMessage += ")";
+
+                _stdout.WriteLine("-" + outputMessage);
+            }
 
-            IGitTfsRemote tfsRemote = null;
+            IGitTfsRemote branchTfsRemote = null;
```

Comment:
Ok. I'm not opposed to this particular renaming. From where the this variable is used, it didn't seem to connote a significant difference to me. But I have no strong objections against this renaming. :smile: